test/java/com/arpnetworking/kairos/service/KairosDbServiceImplTest.java

```diff
@@ -78,6 +80,8 @@
     private Metrics _mockMetrics;
     @Mock
     private MetricsQueryConfig _mockQueryConfig;
+    @Mock
+    private Consumer<MetricsQuery> _mockRewrittenMetricsQueryConsumer;
```

Comment:
This was added because the controller tests were using this mock, when it should have been in this test suite.